packages/core/src/root_wrapper.rs

```diff
@@ -13,27 +13,26 @@ pub(crate) fn RootScopeWrapper(props: RootProps<VComponent>) -> Element {
         None,
         TEMPLATE,
         Box::new([DynamicNode::Component(
-            fc_to_builder(ErrorBoundary)
-                .children(Element::Ok(VNode::new(
+            fc_to_builder(SuspenseBoundary)
```

Comment:
This was not causing the hydration error, but putting the error boundary inside the suspense boundary makes it possible to resolve the error on the server without javascript instead of waiting and resolving the error on the client